lib/huginn_bigcommerce_product_agent/bigcommerce_product_agent.rb

```diff
@@ -143,6 +143,16 @@ def handle_variants(event)
                     product['name'] = "#{product['name']} (Digital)"
                 end
 
+                # Ignatius Press -- some products have the same name and must be disambiguated.
+                # ...by adding a list of the product types (hardback, paperback, etc.) to their names
+                if options['should_disambiguate'] == "true"
```

Comment:
@John-Rabaey The logic here looks good, but I would do a couple things:

*  The option should be a true boolean value, and not a string of "true"
*  You should add validation for this option
   *  It should be _optional_ (evaluate to _false_ if not provided)
   *   If provided, it should be a boolean

(There's a boolean option example here: https://github.com/5-stones/huginn_record_link_agent/blob/master/lib/huginn_record_link_agent/record_link_agent.rb#L222)

Reply:
Done! Thanks for the help!